werf.yaml

```diff
@@ -843,7 +843,7 @@ args:
   BASE_SCRATCH: {{ .Images.BASE_SCRATCH }}
   SOURCE_REPO: {{ .SOURCE_REPO }}
   CLOUD_PROVIDERS_SOURCE_REPO: {{ .CLOUD_PROVIDERS_SOURCE_REPO }}
-  OBSERVABILITY_SOURCE_REPO: {{ OBSERVABILITY_SOURCE_REPO }}
+  OBSERVABILITY_SOURCE_REPO: {{ .OBSERVABILITY_SOURCE_REPO }}
```

Comment:
Error: template: werfConfig:846: function "OBSERVABILITY_SOURCE_REPO" not defined